nav2_bt_navigator/behavior_trees/parallel.xml

```diff
@@ -0,0 +1,16 @@
+<root main_tree_to_execute="MainTree">
+  <BehaviorTree ID="MainTree">
+    <SequenceStar name="root">
+      <ComputePathToPose endpoints="${endpoints}" path="${path}"/>
+      <Parallel threshold_M="1">
+        <FollowPath path="${path}"/>
+        <Sequence>
```

Comment:
Should we make it a standard practice to name the control nodes?
It helps when trying to understand the logic of the tree.

Reply:
Sounds like a good idea. I couldn't come up with anything that wasn't long, such as "planningAndControlInParallel" or something like that. Perhaps adding some comments would be good.